packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/studio/services/representations/DomainViewTreeDescriptionProvider.java

```diff
@@ -295,6 +295,7 @@ private List<TreeItemContextMenuEntry> getContextMenuEntries() {
                 .labelExpression("Toggle abstract")
                 .preconditionExpression(AQL_SELF_IS_AN_ENTITY)
                 .body(callService.build())
+                .withImpactAnalysis(true)
```

Comment:
If this change is fine with you I'll create the cypress test to check the whole feature on this context menu entry action.

Reply:
We will try not to have a lot of changes like that but it's a small one so it's ok for now